api/src/application/client/client-service.js

```diff
@@ -79,7 +79,10 @@ module.exports = (bookshelf) => ({
     return bookshelf.model('client').forge({ client_id: id }).destroy();
   },
   findById(id) {
-    return bookshelf.model('client').forge({ client_id: id }).fetch();
+    return bookshelf
+      .model('client')
+      .where({ client_id: id })
+      .fetch({ withRelated: ['redirect_uris', 'post_logout_redirect_uris'] });
```

Comment:
This should actually return the entire client object (all relations), the merge from master should resolve that